docs/getting-started.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: getting-started
-title: Getting started
+title: Getting Started
 ---
```

Comment:
It would be good to not imply that Desktop is a crippled experience, since it isn't.

Reply:
9865d5c